runelite-client/src/main/java/net/runelite/client/config/RuneLiteConfig.java

```diff
@@ -80,32 +81,21 @@ default boolean gameAlwaysOnTop()
 	}
 
 	@ConfigItem(
-		keyName = "notificationTray",
-		name = "Enable tray notifications",
-		description = "Enables tray notifications",
+		keyName = "notificationMode",
+		name = "Notification mode",
+		description = "Determines mode of notifications",
 		position = 5
 	)
-	default boolean enableTrayNotifications()
```

Comment:
Is it no longer possible to disable notifications with this?

Reply:
Oh, there should be OFF mode, oops.

Reply:
Done